### DISCUSSION TITLE
polkit leftwm in upscript how ?

### DISCUSSION THREAD

**Discussion by NicTanghe** [leftwm/leftwm#1037] [Q&A]
Helo I`ve been having issues with programs that niormally pop up a window tyo give ur admin password.

I think this should be fixed by having some sort of polkit running but Idont know what to do can anyone help ?

**Comment by TH3-S4LM0N** [comment]
Just pick whichever one looks best off [1.1](https://wiki.archlinux.org/title/Polkit) and [xdg autostart](https://github.com/leftwm/leftwm/wiki/Config#autostart) it or run it in the up script.

**Comment by NicTanghe** [comment]
wow that ws fast.

Wel i asked chat gpt and i ended up having to install dbus-11 and then polkit-gnome.
I`ll ad to startup script now.

**Comment by NicTanghe** [comment]
Hello so It seems that after reinstalling I’m running into the same problem only this time it’s not so easily solved.

Seems i keep running into the issue that the gnome-keyring-deamon isn’t being found even though its running.

gnome-keyring-daemon --replace

**Reply by VuiMuich** [reply]
> Hello so It seems that after reinstalling I’m running into the same problem only this time it’s not so easily solved.
> 
> 
> 
> Seems i keep running into the issue that the gnome-keyring-deamon isn’t being found even though its running.
> 
> 
> 
> gnome-keyring-daemon --replace

You are using `systemd`, right? Probably might be worth writing a custom service.

**Reply by NicTanghe** [reply]
I did a reinstall becouse i broke my system trying to fix this and now its somehwat working.
(mailspring is stil complaining that it is not there but i think it is.)

**Reply by VuiMuich** [reply]
Do you know if other WMs have similar issues? Would there be a replacement for `gnome-keyring-daemon` in general?

**Reply by NicTanghe** [reply]
I ended up installing the non gnome one.

**Comment by NicTanghe** [comment]
I should comeback here and post my strategy.
its the polkit-gnome tends to hide in different locations on different distros and also requires to be installed speratly most of the time even if gnome is already installed and polkit works in gnome.

**Comment by grndplane** [comment]
I run this in my up script. I use a Linux Mint 22.1 Cinnamon base.

# Policy Kit to elevate usr for software install
/usr/lib/policykit-1-gnome/polkit-gnome-authentication-agent-1 &
